src/cascadia/TerminalSettingsEditor/SettingContainerStyle.xaml

```diff
@@ -0,0 +1,65 @@
+﻿<!-- Copyright (c) Microsoft Corporation. All rights reserved. Licensed under
+the MIT License. See LICENSE in the project root for license information. -->
+<ResourceDictionary
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:local="using:Microsoft.Terminal.Settings.Editor"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    mc:Ignorable="d">
+
+    <Style x:Key="SettingContainerHeaderStyle" TargetType="StackPanel">
+        <Setter Property="Orientation" Value="Horizontal"/>
+        <Setter Property="Margin" Value="0,0,0,4"/>
+    </Style>
+
+    <Style x:Key="SettingContainerResetButtonStyle" TargetType="Button">
+        <Setter Property="Margin" Value="5,0,0,0"/>
+        <Setter Property="Height" Value="19"/>
+        <Setter Property="Width" Value="19"/>
+        <Setter Property="Padding" Value="0"/>
+        <Setter Property="Background" Value="Transparent"/>
+    </Style>
+
+    <Style x:Key="SettingContainerFontIconStyle" TargetType="FontIcon">
+        <Setter Property="Foreground" Value="{StaticResource SystemAccentColor}"/>
+        <Setter Property="FontSize" Value="14"/>
+        <Setter Property="FontFamily" Value="Segoe MDL2 Assets"/>
+    </Style>
+
+    <Style x:Key="SettingContainerOverrideTextStyle" TargetType="TextBlock">
+        <Setter Property="FontStyle" Value="Italic"/>
+    </Style>
+
+    <Style TargetType="local:SettingContainer">
+        <Setter Property="Margin" Value="0,24,0,0"/>
+        <Setter Property="Template">
+            <Setter.Value>
+                <ControlTemplate TargetType="local:SettingContainer">
+                    <StackPanel>
+
+                        <!--Header space-->
+                        <StackPanel Style="{StaticResource SettingContainerHeaderStyle}">
+                            <TextBlock Text="{TemplateBinding Header}"/>

```

Comment:
do we need to style this or set automation properties on it such that it reads as a header?

Reply:
actually yes how does this whole thing read to a screen reader?

Reply:
For something like commandline, it reads:
"commandline grouping. Executable used in the profile. Edit. powershell.exe"
-->
"<Header> grouping. <HelpText>. Edit. <current setting value>."

I've verified this behavior across all of the profile settings. The only ones that don't work are the ones for radio buttons :(. Screen reader just reads the checked value.

Reply:
OK what does it say when there's an override in effect??

Reply:
Oh, that didn't read it. Hmm...

Reply:
I'm checking with an a11y contact to see what the proper design here should be. If we can fix this, this would let us close out #8899.

Reply:
Spoke with somebody from the accessibility team. Decided on this solution:
- Header (name of the setting) --> AutomationProperty::Name
- Override Message (overriddes what?) --> AutomationProperty::HelpText
- ToolTip (extra description to setting) --> AutomationProperty::DescriptionText